### DISCUSSION TITLE
Amass Tool ImagePullBackOff

### DISCUSSION THREAD

**Discussion by berbarroso** [secureCodeBox/secureCodeBox#1074] [Q&A] [answered]
Hi, 

I'm having problems when trying to scan a website using Amass tool. The status of the scan I created is ImagePullBackOff. I also try to scan other tools and its working. Is there a need for other method since it is upgraded last 5 days ago?

Thank you in advance!

**Comment by malexmave** [comment, marked as answer]
Hi there,

it seems like the relevant docker image by Amass is missing on Dockerhub (it was also leading to failures in our CI process). We'll try to do a new release ASAP where we pull in the upgrade to a working version, thanks for letting us know that this failure also exists in the released version, not just in our CI.

**Reply by rfelber** [reply]
Hi @berbarroso,
we released a new patch version 3.9.1 to fix this issue 👍🏻 Can yo give it a try?

**Reply by berbarroso** [reply]
Hi,

I'm apologize for not getting back sooner. I've been busy with my internship.

Yes Sir, the Amass is now already working. 

Thank you! :)
